### PR TITLE
View Viability Test Modal

### PR DESCRIPTION
**Pull request by constanzauanini**
- View viability test
TODO: Observations Chart, delete

Screen Shot 2022-09-20 at 11 03 02

### CODE REVIEW

File: src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx [hunk 1]
```diff
@@ -15,15 +15,16 @@ type ViabilityTestingPanelProps = {
   user: User;
   setNewViabilityTestOpened: React.Dispatch<React.SetStateAction<boolean>>;
   setSelectedTest: React.Dispatch<React.SetStateAction<ViabilityTest | undefined>>;
+  setViewViabilityTestModalOpened: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
 export default function ViabilityTestingPanel(props: ViabilityTestingPanelProps): JSX.Element {
-  const { accession, setNewViabilityTestOpened, setSelectedTest } = props;
+  const { accession, setNewViabilityTestOpened, setSelectedTest, setViewViabilityTestModalOpened } = props;
```
Comment (karthikbtf) [inline src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx:22]: i have a similar hook in my database PR, after I merge can you rebase and hook it up?

File: src/utils/viabilityTest.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import strings from 'src/strings';
+
+export const getFullTestType = (testType: 'Lab' | 'Nursery' | 'Cut') => {
```
Comment (karthikbtf) [inline src/utils/viabilityTest.tsx:3]: thanks for creating this

File: src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  const { onClose, open, selectedTest, setNewViabilityTestOpened, setSelectedTest } = props;
+
+  const onCloseHandler = () => {
+    setSelectedTest(undefined);
+    onClose();
+  };
+
+  const openEdit = () => {
+    onClose();
+    setNewViabilityTestOpened(true);
```
Comment (karthikbtf) [inline src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx:27]: unrelated to this PR but I realized we need to do input validation for mandatory fields in the edit modal

**Review by karthikbtf — APPROVED** [review]

File: src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx (hunk 3 above)
Comment (constanzauanini) [reply src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx:27]: Yes, I was planning on adding validations at the end

File: src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx:22]: Yes! I will rebase all prs after you merge

File: src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx (hunk 3 above)
Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx:27]: > Yes, I was planning on adding validations at the end

Sounds good 👍

File: src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx:22]: I did all the rebase, but what is the hook you are referring to?

Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx:22]: oh i meant the callback when user clicks on test in the database - you have already connected that, we are good 👍

**Pull request merged**